src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java

```diff
@@ -1,18 +1,12 @@
 package org.testcontainers.containers;
 
 import com.github.dockerjava.api.command.InspectContainerResponse;
-import io.tarantool.driver.exceptions.TarantoolConnectionException;
 
 import org.testcontainers.containers.exceptions.CartridgeTopologyException;
 import org.testcontainers.images.builder.ImageFromDockerfile;
 
 import java.net.URL;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.concurrent.CompletableFuture;
+import java.util.*;
```

Comment:
Please do not use star imports (set up your IDE to not change this for, say, 100 import lines)